modules/mobile/docs/examples/appointments/cc_appointment_request.yml

```diff
@@ -0,0 +1,54 @@
+data:
+  - type: appointment
+    id: '8a48e8db6d70a38a016d72b354240002'
+    attributes:
+      appointmentType: VA
+      cancelId: null
+      comment: null
+      healthcareProvider: null
+      healthcareService: null
+      location:
+        id: '442'
+        name: Cheyenne VA Medical Center
+        address:
+          street: 2360 East Pershing Boulevard
+          city: Cheyenne
+          state: WY
+          zipCode: '82001-5356'
+        lat: 41.148027
+        long: -104.7862575
+        phone:
+          areaCode: '307'
+          number: '778-7550'
+          extension: null
+        url: null
+        code: null
+      minutesDuration: null
+      phoneOnly: false
+      startDateLocal: 2020-11-02T01:00:00.000-07:00
+      startDateUtc: 2020-11-02T08:00:00.000Z
+      status: SUBMITTED
+      statusDetail: null
+      timeZone: America/Denver
+      vetextId: null
+      reason: 'New Issue'
+      is_covid_vaccine: null
```

Comment:
Working on the docs made me realize this should really be false. I initially set it to nil because I thought there would be cases where it would be true, but I've since learned that's probably not so. It doesn't seem to be a problem for the front end, but I can change this to false in a separate PR.